CHANGELOG.md

```diff
@@ -4,6 +4,99 @@ All notable changes to this project are documented in this file.
 
 ## Head
 
+This release contains breaking changes. We know these can be disruptive, but they were necessary to keep our dependencies up to date and stylelint free of security issues.
+
+The most significant change is that we no longer include PostCSS syntaxes (for languages like SCSS or CSS-in-JS libraries like styled-components) in stylelint. You will need to install these custom syntaxes yourself. **If you use stylelint to lint anything other than CSS, you will likely need to change your [configuration object](https://stylelint.io/user-guide/configure/).**
+
+### Migrating from version 13 of stylelint
+
+If you use stylelint to lint CSS, it's unlikely you'll need to make any changes. If you lint a language like SCSS, we recommend installing and [extending](https://stylelint.io/user-guide/configure#extends) a shared-config that includes the appropriate [PostCSS syntax](https://github.com/postcss/postcss#syntaxes) for you. For example, you can use the [stylelint-config-standard-scss]() shared-config for SCSS.
+
+First, install the shared-config as a dependency:
+
+```console
+npm install --save-dev stylelint-config-standard-scss
+```
+
+Then, update your [configuration object](https://stylelint.io/user-guide/configure/) to use it:
+
+```json
+{
+  "extends": ["stylelint-config-standard-scss"],
+  "rules": {
+    ..
+  }
+}
+```
+
+This shared-config extends stylelint to be compatible with SCSS. It configures the built-in rules for SCSS, and includes the [postcss-scss syntax](https://github.com/postcss/postcss-scss) and [stylelint-scss plugin](https://github.com/kristerkari/stylelint-scss) (a collection of rules specific to SCSS).
+
+Other shared-configs that include an appropriate PostCSS syntax are:
+
+- [stylelint-config-standard-styled-components] ???
+
+If a shared-config isn't available for your choice of language or CSS-in-JS library, then you can install the appropriate [PostCSS syntax](https://github.com/postcss/postcss#syntaxes) yourself. For example, to lint [SugarSS](https://github.com/postcss/sugarss) install the syntax as a dependency:
+
+```console
+npm install --save-dev sugarss
+```
+
+Then, update your configuration object to use it:
+
+```json
+{
+  "customSyntax": "sugarss",
+  "rules": {
+    ..
+  }
+}
+```
+
+Other [PostCSS syntaxes](https://github.com/postcss/postcss#syntaxes) known to be compatible with stylelint include:
+
+- [postcss-scss](https://github.com/postcss/postcss-scss)
+- [postcss-less](https://github.com/webschik/postcss-less)
```

Comment:
Oh, it seems the postcss-less's owner has changed: [shellscape/postcss-less](https://github.com/shellscape/postcss-less)

```suggestion
- [postcss-less](https://github.com/shellscape/postcss-less)
```